nano-win.txt

```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = Nano for Windows
+Version = 5.0.4
+License = GPLv3
+Description = Nano for Windows (nano-win) is the ported, standalone version of GNU nano, a famous, easy-to-use and handy text editor on Linux. To extract, 7-Zip or a similar tool is required. Copy the "nano.exe" file in "i686-w64-mingw32\bin\" directory to the target folder, preferably add the directory to PATH in Environment Variables.
+Category = 12
+URLSite = https://github.com/lhmouse/nano-win
+URLDownload = https://files.lhmouse.com/nano-win/nano-win_9142_v5.0-4-gbbd872ca.7z
```

Comment:
Please specify .zip archive download link, or a link to binary executable installer file.

Reply:
The source shares only ".7z" files. The 5.0.4 version is the newest one that contains x86 executable too. So I included the ".7z" file link in URLDownload. But for the URLSite, I picked the project page on Github. I can change `https://github.com/lhmouse/nano-win` to `https://files.lhmouse.com/nano-win`, if you like.